crates/examples/src/objdump.rs

```diff
@@ -134,7 +134,10 @@ fn dump_parsed_object<W: Write, E: Write>(w: &mut W, e: &mut E, file: &object::F
         Err(err) => writeln!(e, "Failed to parse Mach UUID: {}", err)?,
     }
     match file.build_id() {
-        Ok(Some(build_id)) => writeln!(w, "Build ID: {:x?}", build_id)?,
+        Ok(Some(build_id)) => match std::str::from_utf8(build_id) {
+            Ok(build_id) => writeln!(w, "Build ID: {}", build_id)?,
+            Err(std::str::Utf8Error { .. }) => writeln!(w, "Build ID: {:x?}", build_id)?,
+        },
```

Comment:
It will only be a string by coincidence, and we shouldn't print it as a string even in that case.